connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -410,6 +410,16 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
     assert(sorts.isEmpty)
   }
 
+  private def offsetPushed(df: DataFrame, offset: Int): Boolean = {
```

Comment:
`checkOffsetPushed`

Reply:
we should rename `limitPushed` as well.

Reply:
nvm, this two functions return a boolean. We can rename them if we update them to do the check directly instead of returning boolean.